system/authorized_key.py

```diff
@@ -228,9 +234,17 @@ def parseoptions(module, options):
             for part in parts:
                 if "=" in part:
                     (key, value) = part.split("=", 1)
-                    options_dict[key] = value
+                    # certain options can have multiple entries in the authorized_keys file
+                    if key in OPTIONS_WITH_MULTIPLE_VALUES:
+                        if key in options_dict.keys():
```

Comment:
Python style tip: use `if key in options_dict:` without calling `.keys()`.

`keys()` returns a list in Python 2.  Using it changes a O(1) hash lookup into a linear scan.  There's a famous quote by Larry Wall on this topic.